src/lib/components/Swap/Input.tsx

```diff
@@ -81,11 +81,24 @@ export default function Input({ disabled, focused }: InputProps) {
     return insufficientBalance ? 'error' : undefined
   }, [balance, inputCurrencyAmount, swapInputCurrencyAmount])
 
+  const amount = useMemo(() => {
+    if (disabled) {
+      return ''
```

Comment:
You should ping @ppphhhiiilll and ask what this should show when disabled. Showing nothing may look broken.

Reply:
It actually looks more borked when there is a value here. The text input/output looks interactive, but the field itself is disabled.

Reply:
discussed w/ @ppphhhiiilll and confirmed this behavior